test/test_data/emacs.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "emacs",
+        "version": "28.2",
+        "version_strings": [r"$Id: GNU Emacs 28.2"],
+    },
+    {
+        "product": "emacs",
+        "version": "25.2",
+        "version_strings": [r"$Id: GNU Emacs 25.2"],
+    },
+    {
+        "product": "emacs",
+        "version": "26.1",
+        "version_strings": [r"$Id: GNU Emacs 26.1"],
+    },
+]
+package_test_data = [
+    {
+        "url": "http://ports.ubuntu.com/pool/main/e/emacs25/",
+        "package_name": "emacs25_25.2+1-6_arm64.deb",
+        "product": "emacs",
+        "version": "25.2",
+        "other_products": ["libtiff", "pango"],
```

Comment:
```suggestion
```

So after asking you to add those lines... I'm going to remove them because #2986 and #2985 have been merged to avoid the false positive. 

Going to merge this now and let the tests re-run.